systemtests/src/main/java/io/enmasse/systemtest/platform/Minikube.java

```diff
@@ -34,10 +34,11 @@ protected Minikube(Environment environment) {
             OkHttpClient httpClient = HttpClientUtils.createHttpClient(config);
             // Workaround https://github.com/square/okhttp/issues/3146
             httpClient = httpClient.newBuilder()
-                    .protocols(Collections.singletonList(Protocol.HTTP_1_1))
+                    .protocols(Collections.singletonList(Protocol.HTTP_2))
                     .connectTimeout(environment.getKubernetesApiConnectTimeout())
                     .writeTimeout(environment.getKubernetesApiWriteTimeout())
                     .readTimeout(environment.getKubernetesApiReadTimeout())
+                    .retryOnConnectionFailure(true)
```

Comment:
I'd have been dead against this change when we had the api-server, as I'd have worried it would have masked our own defects.

As the api-server is now gone, I am more  relaxed (slightly). If we got a problem connecting to Kubernetes API server, we'd likely class it as platform or environment so retrying would be the right thing to do.  OTOH  if we see are needing to reconnect often - that's kind of worrying - and this change may mask it.  Mmmm.  Decisions, decision!  @ctron @lulf

Reply:
I think it could mask an issue where we were overloading the OpenShift API due to a bug in our operator (see recent bug/fix I made to address-space-controller, which caused various issues in tests using the API), so I would prefer if we didn't retry.